src/sympc/protocol/falcon/falcon.py

```diff
@@ -15,11 +16,15 @@
 from sympc.protocol.protocol import Protocol
 from sympc.session import Session
 from sympc.session import get_session
+from sympc.store import CryptoPrimitiveProvider
+from sympc.store.exceptions import EmptyPrimitiveStore
 from sympc.tensor import MPCTensor
 from sympc.tensor import ReplicatedSharedTensor
 from sympc.tensor.tensor import SyMPCTensor
 from sympc.utils import parallel_execution
 
+shares_sum = ReplicatedSharedTensor.shares_sum
```

Comment:
Q: what does this do?

Reply:
Considering a tensor object ,it might have different operators for summation(binary-xor , normal rings = add(+),prime = add(+) mod prime,it sums the shares according to the ring_size input.

Reply:
In several places for operation on tensors of RSTensor, we call this method.